scripts/report_scripts/timecourse.Rmd

```diff
@@ -0,0 +1,94 @@
+---
+title: "Timecourse"
+output: html_document
+---
+
+#read in config file
+Read config.yml for the locations of sample sheet, pipeline output and sigmut loc and variants_dir
+```{r, child="_settings.Rmd"}
+```
+
+# dummy data
+```{r libraries}
+library(plotly)
+
+#read in sample names from sample sheet 
+df <- read.table("../../tests/sample_data/test_var_timecourse.csv",sep = "\t",header = T)
+df <- read.table(paste0(variant_dir,"/",test_var_timecourse.csv))
```

Comment:
these relative file names are going to cause trouble.  This should be provided as an argument / parameter.

Reply:
It would also be a better practice to use `file.path(variant_dir, "test_var_timecourse.csv")` rather than pasting using separators. Directory separators can be different in different platforms.

Reply:
so the "/" will be in the argument variant_dir ? I was not sure how the arguments are feeded in .. then I will change it in all scripts

Reply:
there is no need for a trailing `/` in `variant_dir`.  `file.path` takes care of joining the individual file name components to a valid file name.

```R
> file.path("this", "that", "whatever")
[1] "this/that/whatever"
```